### DISCUSSION TITLE
What about we mark it stable?

### DISCUSSION THREAD

**Discussion by fballiano** [OpenMage/magento-lts#3399] [Releases]
So, we hit 6 "rc" releases, we fixed and implemented a gazillion things, what about we merge https://github.com/OpenMage/magento-lts/pull/3397, call it a day and mark it stable?

I think we could attract some good attention with a stable release and who knows.

Note: I'm using rc6 on 4 production stores with no problems.

**Comment by sreichel** [comment]
I'd like to see type hints for all new methods before next stable release.

**Reply by fballiano** [reply]
you said it [here](https://github.com/OpenMage/magento-lts/discussions/3250#discussioncomment-5891705) but nobody acted on that in these 2 months so, since
- it is not blocking nor a bug
- it can be added in any moment 
- the docblock are there and correct for sure
- there's no PR to evaluate

I wouldn't halt the release just for this desiderata.

Since you wrote https://github.com/OpenMage/magento-lts/discussions/3250#discussioncomment-5891705 I already tried to make sure that new PRs had type hints.

**Reply by sreichel** [reply]
Also mentioned here #2748 ...  imho it cant be added in any moment. Changing mehtods signnature later could break something. I'd do it asap - before next release.

**Reply by fballiano** [reply]
Ok if you want to work on that it’s welcome, otherwise we do what we can

**Reply by tmotyl** [reply]
I think this release is big enough. Adding more changes makes upgrade more complicated.
 I believe we can introduce breaking changes later on too.

**Comment by fballiano** [comment]
@tmotyl @ADDISON74 @kiatng @elidrissidev I'd still release the last rc version, then not merge anything for a week (unless strictly necessary) and the tag the stable version. is that ok with you?

**Reply by addison74** [reply]
It's been about 5 months since the last stable release. The proposal is fine to me.

**Reply by elidrissidev** [reply]
I'm OK with that.

**Reply by kiatng** [reply]
Thank you sir, for the tireless work. If there is an instability, I'm confident it'll be fixed fast. So, I'm ok.

**Reply by fballiano** [reply]
@ADDISON74 @kiatng @elidrissidev let's go with the release?

**Reply by addison74** [reply]
I took a look at the Issues and PR sections. I didn't find anything notable that would prevent us from releasing a new stable version. From my side it is OK to get a new version after 6 months.
